app/helpers/claims/support/claims_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Claims::Support::ClaimsHelper
+  def claim_draft_state(claim)
+    css_class = claim.draft ? "govuk-tag--grey" : "govuk-tag--blue"
+    text = claim.draft ? "Draft" : "Submitted"
+
+    "<strong class='govuk-tag #{css_class}'>#{text}</strong>".html_safe
```

Comment:
Do you think we can use `contet_tag` for this html? You shouldn't need to call html_safe with a content tag I think https://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag